docs/tutorial/gatsby-image-tutorial/index.md

```diff
@@ -91,6 +91,8 @@ export const query = graphql`
 `
 ```
 
+_**Note:** This query won't work in GraphiQL as it doesn't support implicit fragments. This will work in your Gatsby pages and components. [See the `gastby-image` docs for more details](https://www.gatsbyjs.org/packages/gatsby-image/#fragments)._
```

Comment:
Typo

```suggestion
_**Note:** This query won't work in GraphiQL as it doesn't support implicit fragments. This will work in your Gatsby pages and components. [See the `gatsby-image` docs for more details](https://www.gatsbyjs.org/packages/gatsby-image/#fragments)._
```